widget/playground/src/components/ExportConfigModal/ExportConfigModal.tsx

```diff
@@ -0,0 +1,157 @@
+import type {
+  ExportConfigModalProps,
+  ExportType,
+} from './ExportConfigModal.types';
+
+import {
+  CloseIcon,
+  Divider,
+  ExternalLinkIcon,
+  Modal,
+  ModalHeader,
+  TextField,
+  Typography,
+} from '@rango-dev/ui';
+import React, { Fragment, useState } from 'react';
+import {
+  atomDark as dark,
+  prism,
+} from 'react-syntax-highlighter/dist/esm/styles/prism';
+
+import {
+  capitalizeTheFirstLetter,
+  filterConfig,
+  formatConfig,
+} from '../../helpers';
+import { useTheme } from '../../hook/useTheme';
+import { initialConfig } from '../../store/config';
+
+import { CodeBlock } from './CodeBlock';
+import {
+  APIKeyInputContainer,
+  ButtonsContainer,
+  ExternalLinkIconContainer,
+  Flex,
+  HelpLinksContainer,
+  Link,
+  LinkContainer,
+  ModalFlex,
+  StyledButton,
+  StyledIconButton,
+} from './ExportConfigModal.styles';
+import { typesOfCodeBlocks } from './ExportConfigModal.types';
+
+export function ExportConfigModal(props: ExportConfigModalProps) {
+  const { open, onClose, config } = props;
+
+  const { activeTheme } = useTheme();
+  const [selected, setSelected] = useState<ExportType>('embedded');
+  const syntaxHighlighterTheme = activeTheme === 'dark' ? dark : prism;
+  const [apiKey, setApiKey] = useState<string>(initialConfig.apiKey);
+  const { filteredConfigForExport } = filterConfig(
+    config,
+    initialConfig,
+    apiKey
+  );
+  const formatedConfig = formatConfig(filteredConfigForExport);
+
+  return (
+    <Modal
+      containerStyle={{
+        width: '1109px',
+        height: '865px',
+        padding: '$20',
+        fontFamily: 'Roboto',
```

Comment:
Shouldn't we read the font from `theme`?

Reply:
Good point. But what I know, we are going to apply the widget theme to our playground as well. if we are going to override some parts of that (e.g. font), I suggest to define the value inside `widget/playground/src/constants.ts` and use it here, it lets us to easily change them in future.

Reply:
Yes, but it seems we didn't go this way in previous PRs.  But I moved it to constant as you suggested